src/routes/consent.js

```diff
@@ -2,6 +2,10 @@ module.exports = {
   method: 'GET',
   path: '/consent',
   handler: function (request, reply) {
-    reply.view('consent')
+    let data = {
+      url: 'https://www.youtube.com/embed/bGIhghnseYk?rel=0&amp;autoplay=1&amp;controls=0&amp;showinfo=0',
```

Comment:
Hmm, okay can you make a file called video urls.
```js
src/utils/video_urls.js

module.exports = {
  help: 'https://'
  comeAgain: ' ...',
  ...
}
```

I think this may be better if you need to change them all ..

If you would rather not then thats fine its not important